### ISSUE TITLE
[tuify][giti] Release it and publicize it on twir, r/rust

### ISSUE THREAD

**Issue by nazmulidris** [r3bl-org/r3bl-open-core#216]

**Comment by Harshil-Jani** [comment]
@nazmulidris Please do let me know about when to do this. Probably on Saturday I can write a spicy blog on tuify and refer giti as example in blog post and publish the link into TWiR.

**Comment by nazmulidris** [comment]
@NadiaIdris @Harshil-Jani Maybe the two of you can coordinate on this and take care of creating the content & publicizing it.

**Comment by NadiaIdris** [comment]
@Harshil-Jani Yes, please, go for it 🙂! Let me know when it's ready for proof reading or if you want to share an early draft, that's cool too. If you need any help with it, let me know. We can publish it on developerlife.com.
